python/ray/experimental/sgd/tf/tf_trainer.py

```diff
@@ -92,24 +95,65 @@ def __init__(self,
                 for i, worker in enumerate(self.workers)
             ])
 
-    def train(self):
+        import tensorflow as tf
+        self._Progbar = tf.keras.utils.Progbar
+
+    def _generic_model_driver(self, request_step, steps, metrics_prefix):
         """Runs a training epoch."""
```

Comment:
Can you also add docstring? https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html